docs/source/tutorial/01-configure-project.mdx

```diff
@@ -8,33 +8,34 @@ In this step, you'll import the starter project and add the Apollo Kotlin SDK to
 
 ```
 git clone https://github.com/apollographql/apollo-kotlin-tutorial
-cd apollo-kotlin-tutorial
-git checkout initial
+cd apollo-kotlin-tutorial/compose
 ```
 
-At any time, if you want to view a completed version of the project, checkout the `main` branch.
+The starter project is in the `initial` folder. 
+
+At any time, if you want to view a completed version of the project, open the project in the `finished` folder.
```

Comment:
Nit: I like `final` as the opposite of `initial` but I'm no native speaker so might be wrong about it.

Reply:
Agree `finished` is not great. How about `completed`?

Reply:
In the fullstack tutorial we use `start` and `final` which have worked great! Could probably roll with those 😄